Orange/canvas/document/editlinksdialog.py

```diff
@@ -695,23 +768,32 @@ def setChannel(self, channel):
         if channel != self.__channel:
             self.__channel = channel
 
-            if hasattr(channel, "description"):
-                self.setToolTip(channel.description)
-            # TODO: Should also include name, type, flags, dynamic in the
-            #       tool tip as well as add visual clues to the anchor
-
     def channel(self):
         """
         Return the channel description.
         """
         return self.__channel
 
+    def setEnabled(self, enabled):
+        QGraphicsRectItem.setEnabled(self, enabled)
+        if enabled:
+            self.setBrush(self.enabledBrush)
+        else:
+            self.setBrush(self.disabledBrush)
+
+    def paint(self, painter, option, widget=None):
+        QGraphicsRectItem.paint(self, painter, option, widget)
+        # if disabled, draw X over box
+        if not self.isEnabled():
+            painter.drawLine(0, 0, int(self.rect().width()), int(self.rect().height()))
+            painter.drawLine(int(self.rect().width()), 0, 0, int(self.rect().height()))
```

Comment:
```python
painter.drawLine(self.rect().topLeft(), self.rect().bottomRight())
painter.drawLine(self.rect().topRight(), self.rect().bottomLeft())
```
ChannelAnchor is a QGraphicsRectItem, it does not necessarily have (0, 0) as its top right corner.